dataconnect/README.md

```diff
@@ -0,0 +1,73 @@
+3. Select your server region, if you wish to use vector search, make sure to select `us-central1` region.
+4. Allow some time for the Cloud SQL instance to be provisioned. After it's provisioned, the instance can be managed in the [Cloud Console](https://pantheon.corp.google.com/sql).
+
+### 2. Set Up Firebase CLI
+
+Ensure the Firebase CLI is installed and up to date:
+
+```bash
+npm install -g firebase-tools
+```
+
+### 3. Cloning the repository
+This repository contains the quickstart to get started with the functionalities of Data Connect.
+
+1. Clone this repository to your local machine.
+2. Enable the DataConnect CLI with the command `firebase experiments:enable dataconnect`.
+3. cd to the `dataconnect` folder, and initialize your Firebase project with `firebase init` and select both DataConnect options. Overwrite only dataconnect.yaml when prompted, do not overwrite any other dataconnect files.
+4. Replace variables in `.env` with your project-specific values.
+5. Allow domains for Firebase Auth in your [project console](https://console.firebase.google.com/project/_/authentication/settings) (e.g. http://127.0.0.1).
+
+### 4. Running queries and mutations in VS Code
+The VSCode Firebase Extension allows you to generate Firebase Data Connect SDK code, run queries/mutations, and deploy Firebase Data Connect with a click. Alternatively, see below for CLI commands.
+
+1. Install [VS Code](https://code.visualstudio.com/).
+2.  Download the [Firebase extension](https://firebasestorage.googleapis.com/v0/b/firemat-preview-drop/o/vsix%2Ffirebase-vscode-latest.vsix?alt=media) and [install](https://code.visualstudio.com/docs/editor/extension-marketplace#_install-an-extension) it.
+3. Open this quickstart in VS code, and in the left pane of the Firebase extension, and log in with your Firebase account.
+(Optional): If your Firebase project was not initialized in the last section, you can click `Run firebase init` and select `Data Connect` to initialize.
+4. Click on deploy to deploy your schema to your cloud SQL instance. Or run `firebase deploy --only dataconnect` (this will also activate vectors search if it's enabled in the schema).
+4. Running the VSCode extension should automatically start the DataConnect emulators. If you see an emulators error, try running `firebase emulators:start` manually.
+
+Now you should be able to deploy your schema, run mutations/queries, generate SDK code, and view your application locally.
+
+### 5. Populating the database
+1. Run the four `_insert.gql` files in the `./dataconnect` directory in order using the VS code extension.
+
+### 6. Running the app
+
+1. `cd` into the src folder, and run `npm run dev` to start the localhost web app.
+(Optional): To use the Firebase Auth emulator, run `firebase emulators:start` and uncomment the `connectAuthEmulator` function in `src/lib/firebase.ts`.
```

Comment:
Is this still accurate?

I see `src/lib/firebase.tsx` instead. Couldn't find `connectAuthEmulator` in https://github.com/firebase/quickstart-js/pull/814/files#diff-9728a04df367a801a8d0f63192f63d3680e1e75b114444a51a75a1876b3d8424

Reply:
Good catch, it got lost in process of migration... should be added back now!